pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -2853,6 +2853,22 @@ let
 
       ms-python.python = callPackage ./ms-python.python { };
 
+      ms-python.debugpy = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "debugpy";
+          publisher = "ms-python";
+          version = "2023.3.13121011";
+          hash = "sha256-owYUEyQl2FQytApfuI97N4y9p7/dL0lu6EBk/AzSMjw=";
+        };
+        meta = with lib; {
+          description = " Python debugger (debugpy) extension for VS Code";
+          downloadPage = "https://marketplace.visualstudio.com/items?itemName=ms-python.debugpy";
+          homepage = "https://github.com/Microsoft/vscode-python-debugger";
+          license = licenses.mit;
+          maintainers = with maintainers; [ carlthome ];
```

```suggestion
        meta = {
          description = "Python debugger (debugpy) extension for VS Code";
          downloadPage = "https://marketplace.visualstudio.com/items?itemName=ms-python.debugpy";
          homepage = "https://github.com/Microsoft/vscode-python-debugger";
          license = lib.licenses.mit;
          maintainers = [ lib.maintainers.carlthome ];
```
Without an extra space at the beginning of the description and `with` statement